test/relay.test.js

```diff
@@ -136,10 +137,24 @@ describe('relay', function () {
       }
     });
 
+    viewerType = new GraphQLObjectType({
+      name: 'Viewer',
+      description: 'root viewer for queries',
+      fields: () => ({
+        id: globalIdField('Viewer'),
+        name: {
+          type: GraphQLString,
+          resolve: () => 'Viewer!'
+        }
+      }),
+      interfaces: [nodeInterface]
+});
+
     nodeTypeMapper.mapTypes({
       [User.name]: userType,
       [Project.name]: projectType,
-      [Task.name]: taskType
+      [Task.name]: taskType,
+      viewer: viewerType
```

Comment:
I'm not totally happy with the uncased matching, but i doubt people willa ctually have types with different cases of the same name - But if it does happen we'll break it later.